etc/checkfmt.sh

```diff
@@ -12,39 +13,39 @@ for file in $(go list -f '{{$dir := .Dir}}{{range .GoFiles}}{{printf "%s/%s\n" $
         # Only print this once.
         HAS_FMT_ERR=1
         echo 'Commit includes files that are not gofmt-ed' && \
-        echo 'run "make fmt"' && \
         echo ''
       fi
-      echo "$FMT_OUT" # Print output and continue, so developers don't fix one file at a t
+      echo "$FMT_OUT" # Print output and continue, so developers don't fix one file at a time
     fi
    fi
 done
 
-## print at the end too... sometimes it is nice to see what to do at the end.
-if [ "$HAS_FMT_ERR" -eq "1" ]; then
-    echo 'Commit includes files that are not gofmt-ed' && \
-    echo 'run "make fmt"' && \
-    echo ''
-fi
-
 (
-  cd libflux || exit
-  FMT_OUT="$(cargo fmt --all -- --check)"
-  if [[ -n "$FMT_OUT" ]]; then
+    echo "Checking Rust source files..."
```

Comment:
Indentation seems to be a bit weird here.

Reply:
This bit was previously wrapped in `()` so I just left that part alone.

Reply:
The weird part is that `echo` is 4 spaces and `cd` is 2 spaces of indentation.